src/app.scss

```diff
@@ -64,6 +64,9 @@ html, body {
   display:flex;
   justify-content:center;
 }
+.loader {
+  margin: 40%;
+}
```

Comment:
I think it is better to create a wrapper around AeLoader than add stuff to global styles.